app/src/marketplace/components/ProductTypeChooser/index.jsx

```diff
@@ -40,7 +40,7 @@ const ProductTypeChooser = ({ onSelect }: Props) => (
                     <button
                         type="button"
                         className={cx('btn', 'btn-special', styles.button)}
-                        onClick={() => onSelect('standard')}
+                        onClick={() => onSelect('NORMAL')}
```

Comment:
I can't remember if it was spec to leave out type for regular products or not?

Reply:
We can leave it out in the service if needed. I think the backend code will default to `type = normal` if it is left out. I think it's good to be explicit if backend supports this.

Reply:
Yeah, let's go with this 👍